<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>541</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>回显模式</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>521</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnEchoNormal">
       <property name="text">
        <string>正常模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEchoPassword">
       <property name="text">
        <string>密码模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEchoNoEcho">
       <property name="text">
        <string>无回显</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEchoPasswordEchoOnEdit">
       <property name="text">
        <string>失去焦点变密码模式</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>521</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="leEcho"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnEchoPrint">
       <property name="text">
        <string>输出内容</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>对齐方式</string>
   </property>
   <widget class="QLineEdit" name="leAlignment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>521</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnAlignLeft">
       <property name="text">
        <string>左对齐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAlignCenter">
       <property name="text">
        <string>居中对齐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAlignRight">
       <property name="text">
        <string>右对齐</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>读写控制</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>521</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnAccessReadWrite">
       <property name="text">
        <string>可读可写</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAccessReadOnly">
       <property name="text">
        <string>只读</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAccessDisabled">
       <property name="text">
        <string>禁能</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAccessEnabled">
       <property name="text">
        <string>使能</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>521</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="leAccess"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAccessPrint">
       <property name="text">
        <string>输出内容</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>格式控制</string>
   </property>
   <widget class="QLineEdit" name="leMask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>521</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnMaskNone">
       <property name="text">
        <string>无控制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMaskPhone">
       <property name="text">
        <string>手机号码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMaskLicense">
       <property name="text">
        <string>秘钥</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>信号槽</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>155</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>    textChanged信号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>155</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>editingFinished信号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAdd1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAdd2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAddResult">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSub1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSub2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSubResult">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
